go/cmd/dolt/commands/mysql_rows_iter.go

```diff
@@ -0,0 +1,278 @@
+package commands
+
+import (
+	dsql "database/sql"
+	"fmt"
+	"github.com/dolthub/go-mysql-server/sql"
+	"github.com/dolthub/go-mysql-server/sql/parse"
+	"io"
+	"reflect"
+	"strings"
+)
+
+// MySqlRowsIter is an iterator over rows returned by MySQL
+type MySqlRowsIter struct {
+	rows        *dsql.Rows
+	types       []sql.Type
+	columnTypes []*dsql.ColumnType
+	schema      sql.Schema
+}
```

Comment:
Where is 'types' ever use once the object is constructed? It looks like you never use it, so I wouldn't bother putting it in the struct.